app/serializers/export/contact_serializer.rb

```diff
@@ -1,12 +1,28 @@
 module Export
   class ContactSerializer < ActiveModel::Serializer
     attributes :sharing,
-               :receiving,
+               :following,
                :person_guid,
                :person_name,
-               :person_first_name,
-               :person_diaspora_handle
+               :account_id,
+               :serialized_public_key
```

Comment:
Oh, another little thing: You renamed `serialized_private_key` to `private_key` for the users own private key, So I think this should be renamed to just `public_key`.